Cargo.toml

```diff
@@ -24,6 +24,7 @@ smallvec = "1.7"
 aes = { version = "0.8.3", features = ["hazmat"] }
 measureme = "11"
 ctrlc = "3.2.5"
+chrono = { version = "0.4.38", features = ["std"] }
```

Comment:
you need to set `no-default-features=true`, otherwise all default features are still enabled, not just the ones you explicitly turned on